packages/cli/generators/project/templates/tslint.build.json

```diff
@@ -1,17 +1,4 @@
 {
   "$schema": "http://json.schemastore.org/tslint",
-  "extends": ["./tslint.json"],
-  // This configuration files enabled rules which require type checking
-  // and therefore cannot be run by Visual Studio Code TSLint extension
-  // See https://github.com/Microsoft/vscode-tslint/issues/70
-  "rules": {
-    // These rules find errors related to TypeScript features.
-
-    // These rules catch common errors in JS programming or otherwise
-    // confusing constructs that are prone to producing bugs.
-
-    "await-promise": true,
-    "no-floating-promises": true,
-    "no-void-expression": [true, "ignore-arrow-function-shorthand"]
-  }
+  "extends": ["./node_modules/@loopback/build/config/tsconfig.build.json"]
```

Comment:
I think you have a typo here - are you sure that `tslint.build.json` should be extending `tsconfig.build.json`? I think you meant to extend `tslint.build.json` instead.

Reply:
Good catch. Fixed.